root/static/scripts/common/components/TagEditor.js

```diff
@@ -37,6 +37,15 @@ const cmpTags = (a, b) => (
   (b.count - a.count) || compareStrings(a.tag.name, b.tag.name)
 );
 
+function formatGenreLabel(genre) {
+  if (!genre.comment) {
+    return genre.name;
+  }
+
+  return genre.name + '<span class="small"> ' +
+         bracketedText(genre.comment) + '</span>';
```

Comment:
(you'll have to import the he module here)
```suggestion
  return he.encode(genre.name) +
    '<span class="small"> ' +
    he.encode(bracketedText(genre.comment)) +
    '</span>';
```

Reply:
Never mind, made the change!